recipes/glog/all/conanfile.py

```diff
@@ -63,6 +63,14 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.components["libglog"].libs = tools.collect_libs(self)
+        self.cpp_info.components["libglog"].names["pkgconfig"] = ["libglog"]
         if self.settings.os == "Linux":
-            self.cpp_info.system_libs.append("pthread")
+            self.cpp_info.components["libglog"].system_libs = ["pthread"]
+        elif self.settings.os == "Windows":
+            self.cpp_info.components["libglog"].defines = ["GLOG_NO_ABBREVIATED_SEVERITIES"]
+            if self.settings.compiler == "Visual Studio":
+                decl = "__declspec(dllimport)" if self.options.shared else ""
+                self.cpp_info.components["libglog"].defines.append("GOOGLE_GLOG_DLL_DECL={}".format(decl))
```

Comment:
Does it work if Windows and not Visual Studio?

Reply:
I didn't test, but according upstream's cmake file, there is no restriction. I gonna update it and let's see.

Reply:
Now is allowed for any compiler on Windows, just like the upstream.